src/libraries/System.DirectoryServices/src/System/DirectoryServices/ActiveDirectory/Utils.cs

```diff
@@ -1935,10 +1935,10 @@ internal static string SplitServerNameAndPortNumber(string serverName, out strin
             }
 
             //extract IPv6 port number if any
-            bool isBrace = serverName.StartsWith("[");
+            bool isBrace = serverName.StartsWith("[", StringComparison.Ordinal);
```

Comment:
Is this netstandard2.0 and that's why we can't just pass '['?

Reply:
Yeah, unfortunately. :(
https://github.com/dotnet/runtime/blob/fcfb4b8a62b0bc30d9ed83a08cf9b322908c3973/src/libraries/System.DirectoryServices/src/System.DirectoryServices.csproj#L7

Reply:
Why is it netstandard at all?

Reply:
Because it's part of the Windows Compat Pack, which shipped for .NET Standard 2.0/.NET Standard 2.0. We decided that we'd rather have a turd assembly then having to deal with disjoint API surface area between Windows and non-Windows (i.e. a throwing is API is better than an API that is missing based on operating system).

I don't think we can change this for .NET 5 because customers building for .NET Standard 2.0 are expected to not lose any APIs when retargeting to `net5`.